kubernetes/product-deployment/main.jenkins

```diff
@@ -0,0 +1,286 @@
+				text-align: center;
+				padding: 4px;
+				color: #343434;
+				font-size: 22px;
+				font-weight: normal;
+				text-transform: uppercase;
+				font-family: 'Orienta', sans-serif;
+				letter-spacing: 1px;
+				font-style: italic;
+                margin-bottom: 25px;
+			"""),
+                            string(
+                                name: 'deployment_script_repo_url',
+                                defaultValue: 'https://github.com/Tharsanan1/APIMCloudFormation',
+                                description: 'Cloud formation repository url.',
+                                trim: true
+                            ),
+                            string(
+                                name: 'deployment_script_repo_branch',
+                                defaultValue: 'main',
+                                description: 'Branch name of the cloud formation repo.',
+                                trim: true
+                            ),
+                            string(
+                                name: 'db_engine',
+                                defaultValue: 'mysql',
+                                description: 'Database engine',
+                                trim: true
+                            ),
+                            string(
+                                name: 'db_version',
+                                defaultValue: '8.0.28',
+                                description: 'Database engine version',
+                                trim: true
+                            ),
+                            string(
+                                name: 'db_instance_class',
+                                defaultValue: 'db.t3.small',
+                                description: 'Database engine instance class',
+                                trim: true
+                            ),
+                            separator(name: "KUBERNETES_ENVIRONMENT", sectionHeader: "Kubernetes Details",
+                                separatorStyle: "border-width: 0",
+                                sectionHeaderStyle: """
+                                    background-color: #7ea6d3;
+                                    text-align: center;
+                                    padding: 4px;
+                                    color: #343434;
+                                    font-size: 22px;
+                                    font-weight: normal;
+                                    text-transform: uppercase;
+                                    font-family: 'Orienta', sans-serif;
+                                    letter-spacing: 1px;
+                                    font-style: italic;
+                                    margin-bottom: 25px;
+                                """),           
+                            string(
+                                name: 'kubernetes_repo_url',
+                                defaultValue: 'https://github.com/Tharsanan1/kubernetes-apim',
```

Comment:
shall we remove this value from default as these are folked repos? WDYT @VimukthiPerera

Reply:
+1